internal/controller/pkg/revision/runtime_test.go

```diff
@@ -353,30 +328,12 @@ func TestRuntimeManifestBuilderDeployment(t *testing.T) {
 					namespace: namespace,
 				},
 				serviceAccountName: functionRevisionName,
-				overrides:          functionDeploymentOverrides(&pkgmetav1beta1.Function{}, functionRevision),
+				overrides:          functionDeploymentOverrides(functionImage),
 			},
 			want: want{
 				want: deploymentFunction(functionName, functionRevisionName, functionImage),
 			},
 		},
-		"FunctionDeploymentWithImageOverride": {
```

Comment:
the logic that this test was covering now actually lives at a higher level - this logic is now covered by `TestGetFunctionImage`